public/css/primitives.css

```diff
@@ -7,8 +7,8 @@
 	--last-period-warning-color: #c78a27;
 	--unknown-color: #dab092;
 	--notstarted-color: #97A0AF; /* N80 - Spooky ghost */
-	--blocked-color: rgb(242, 0, 11);
-	--last-period-blocked-color: rgb(176, 40, 47);
+	--blocked-color: #f2000b;
+	--last-period-blocked-color: #b0282f;
```

Comment:
this is the same color but in hex, for some reason vscode wasn't showing me the color with RGB like it does with hex, can change it back if needed